collector/hadoop/Makefile

```diff
@@ -9,4 +9,9 @@ wordcount:
 	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} hadoop-wordcount
 	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} bde2020/hadoop-namenode:$(tag) hdfs dfs -cat /output/*
 	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} bde2020/hadoop-namenode:$(tag) hdfs dfs -rm -r /output
-	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} bde2020/hadoop-namenode:$(tag) hdfs dfs -rm -r /input
\ No newline at end of file
+	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} bde2020/hadoop-namenode:$(tag) hdfs dfs -rm -r /input
+
+cleanup:
```

Comment:
This command makes irrecoverable changes to the system beyond the scope of the example.

Did you consider the solutions that I put in my comment? Seems like either of those should work, and they'd have `docker compose down.` 

If `docker compose exec` can work then you never create additional containers in the first place. That one seems it would be best, because I think there's a significant startup penalty with this container.

If that won't work then you just add the `--rm` flag in your command and it will automatically cleanup when the container shuts down.

Reply:
@nslaughter Addressed your comments by adding `docker compose down` to instructions in the readme. FYI, `docker run --rm`  is not working for later versions (20.10) of Docker on a Mac.  It might work for you if you have older version installed.

Reply:
@hecomp I wasn't able to reproduce any such failure of the `--rm` flag with a current version of Docker desktop. Did you find a related bug report on this or file one?

Reply:
I think this would simplify a lot of we just threw out all parts related to the example job. Hadoop is wired up. The example doesn't work.

Reply:
> @hecomp I wasn't able to reproduce any such failure of the `--rm` flag with a current version of Docker desktop. Did you find a related bug report on this or file one?

@nslaughter may be is my environment. I did looked into it for a particular report or release from Docker about this issue in my side. I've found example usage of the flag but didn't show that worked like mine.